featurebyte/query_graph/node/input.py

```diff
@@ -369,17 +369,6 @@ class InputNode(BaseNode):
         TableDataType.SCD_TABLE: ClassEnum.SCD_TABLE,
     }
 
-    @root_validator(pre=True)
-    @classmethod
-    def _set_default_table_data_type(cls, values: Dict[str, Any]) -> Dict[str, Any]:
-        # DEV-556: set default table type when it is not present
-        if values.get("type") == NodeType.INPUT:
-            # only attempt to fix this if it is an INPUT node
-            # otherwise, it may cause issue when deserializing the graph in fastapi response
-            if "parameters" in values and "type" not in values["parameters"]:
-                values["parameters"]["type"] = TableDataType.EVENT_TABLE
-        return values
```

Comment:
This part is removed as replacing `BaseModel` with `FeatureByteBaseModel` causes errors. Since we have introduced `parameters.type` quite some time, should be safe to remove it.